zbus/src/message/header.rs

```diff
@@ -353,7 +358,7 @@ impl<'m> Header<'m> {
     }
 }
 
-static SERIAL_NUM: AtomicU32 = AtomicU32::new(1);
+static SERIAL_NUM: AtomicU32 = AtomicU32::new(0);
```

Comment:
Actually we don't want this. Traditionally all serial numbers start from 1 and since there is no reason not to start with a 1, let's continue doing that. I can imagine this also causing people to think that some message was lost when monitoring the bus traffic for troubleshooting.

Reply:
Oh, I misread. I'm sorry. This is actually fixing the issue of not starting with a 1. :laughing: